code/game/machinery/firealarm.dm

```diff
@@ -11,6 +11,7 @@
 	icon = 'icons/obj/monitors.dmi'
 	icon_state = "fire_bitem"
 	result_path = /obj/machinery/firealarm
+	pixel_shift = 26
```

Comment:
Previous pixel "shifting" was 24, why change it to 26?

Reply:
I thought it looked off.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/577cec70-8373-41db-b0f7-7153b8e20933)

I think they should be just below air alarms, but a bit higher than they were. Just a consideration for possible wallening in the future.